### ISSUE TITLE
Error when run model-analyzer in remote mode

### ISSUE THREAD

**Issue by badskeet** [triton-inference-server/model_analyzer#263]
I'm trying to run model-analyzer remotely for my custom model but it is failing with the following error:
 
`2021-12-03 20:16:15.587 ERROR[entrypoint.py:214] Model Analyzer encountered an error: Failed to set the value for field "triton_server_path". Error: Either the binary 'tritonserver' is not on the PATH, or Model Analyzer does not have permissions to execute os.stat on this path.`
 
I cannot specify the `triton_server_path` because I use remote mode model-analyzer and run tritonserver and model-analyzer in different containers.
I do next steps to reproduce the error:
 
1. Run tritonserver container on custom ports.
2. Run sdk container and then run command:
 
`Singularity> model-analyzer profile --model-repository=/models --profile-models=detector_trt  --triton-launch-mode=remote --triton-http-endpoint=localhost:5000 --triton-grpc-endpoint=localhost:5001 --triton-metrics-url=localhost:5002`
 
I use singularity to run containers but it does’t matter.

**Comment by Tabrizian** [comment]
@badskeet This issue has been fixed in the main branch of Model Analyzer and will be available for 21.12. As a workaround, could you please set the `triton_server_path` field to a binary file that exists?

**Comment by okdimok** [comment]
Since this issue is still open, I will add to it. After using the [workaround](https://github.com/triton-inference-server/model_analyzer/issues/263#issuecomment-989955833), there is a similar error:
```
2021-12-11 12:47:36.814 ERROR[entrypoint.py:214] Model Analyzer encountered an error: Failed to set the value for field "triton_install_path". Error: Either '/opt/tritonserver' is a nonexistent path, or Model Analyzer does not have permissions to execute os.stat on this path 
```

It can be worked around simply by
```
mkdir -p /opt/tritonserver
```
